sdk/typescript/src/index.ts

```diff
@@ -1,4 +1,3 @@
-export { gql } from "graphql-tag"
```

Comment:
I removed this import because it fails with `Deno` and it's also a useless export, user is free to use any external library to format GQL request.
It's not a breaking changes, there's a very low chance that anybody actually ever used that.